src/main/java/sirius/kernel/commons/Files.java

```diff
@@ -28,6 +29,14 @@ public class Files {
 
     private static final Pattern NON_PATH_CHARACTERS = Pattern.compile("[^a-zA-Z0-9\\-.]");
 
+    /**
+     * Contains a list of file names and endings which are considered to be metadata.
+     */
+    private static final List<String> METADATA_FILES = List.of(
+            "__MACOSX",
+            ".DS_Store",
+            "thumbs.db");
```

Comment:
I think Windows calls it `Thumbs.db`, (uppercase T). At least is what I find in some sample in our Google Drive storage.